src/index.js

```diff
@@ -1,18 +1,33 @@
-import React, { lazy, Suspense } from 'react';
+import React, { lazy, Suspense, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
 import { Switch } from 'react-router-dom';
-import { Route, coreEvents, HandlerManager } from '@folio/stripes/core';
+import { AppContextMenu, Route, coreEvents, HandlerManager } from '@folio/stripes/core';
+
+import {
+  CommandList,
+  HasCommand,
+  KeyboardShortcutsModal,
+  NavList,
+  NavListItem,
+  NavListSection,
+  checkScope,
+  defaultKeyboardShortcuts,
+} from '@folio/stripes/components';
 
 import PropTypes from 'prop-types';
 import Registry from './Registry';
 
+import DashboardsRoute from './routes/DashboardsRoute';
+import DashboardRoute from './routes/DashboardRoute';
+import DashboardOrderRoute from './routes/DashboardOrderRoute';
+import WidgetCreateRoute from './routes/WidgetCreateRoute';
+
 const Settings = lazy(() => import('./settings'));
```

Comment:
Any reason this is left out as a dynamic imoprt?

Reply:
it's not causing issues as the others, so I left it as it was

Reply:
Yeah, I mean just for consistency lets take it out. We can add it back when we actually have code splitting fully supported. Refer to the comment here https://folio-project.slack.com/archives/CAN13SWBF/p1580423284014600 for more information.